server/src/server.rs

```diff
@@ -587,6 +558,8 @@ pub(crate) struct ServiceData<L: Logger> {
 	pub(crate) enable_http: bool,
 	/// Enable WS.
 	pub(crate) enable_ws: bool,
+	/// Bounded channel capacity.
+	pub(crate) buffer_capacity: u32,
```

Comment:
Do we need the `buffer_capacity` on this structure for the moment?

Reply:
Yeah, it's the number of messages that server are allowed to have in memory.

we could rename this if it's unclear however we could remove `max request` and `max response` but kept that for now as the messages could be of arbitrary size.